RELEASENOTES.txt

```diff
@@ -4,6 +4,73 @@ order.
 For details on installing PAPI on your machine, consult the INSTALL.txt file
 in this directory.
 
+
+===============================================================================
+PAPI 7.2.0 RELEASE NOTES                                               Jun 2025
+===============================================================================
+
+PAPI 7.2.0 is now available as the next major release. This release officially
+introduces two new components:
+  +++ rocp_sdk: Supports AMD GPUs and APUs via the ROCprofiler-SDK interface.
+  +++ topdown: Provides proper support for Intel topdown metrics.
+PAPI 7.2.0 also introduces preset events for non-CPU devices, starting with
+CUDA events. In addition, component code has been extended to include a
+statistics qualifier (e.g., for CUDA events), offering more concise and
+functional output in the papi_native_avail utility.
+
+
+Additional Major Changes are:
+-----------------------------
+
+Component Updates:
+------------------
+
+* RAPL: Support for Intel Emerald Rapids, and Intel Comet Lake S/H CPUs.
+
+* ROCM/ROCP_SDK:
+  +++ Numerous improvements to error handling, shutdown behavior, and
+      initialization in `rocm` and `rocp_sdk` components.
+  +++ Added multiple libhsa search paths.
+  +++ Correct handling if all events are removed.
+  +++ Improved interoperability between `rocm` and `rocp_sdk` components.
+
+* CUDA:
+  +++ Added statisitcs qualifier to CUDA events, which offers a more concise
+      and functional output for the papi_native_avail utility.
+  +++ Added `partially enabled` support for systems with multiple compute
+      capabilities: <7.0, =7.0, >7.0.
+  +++ Support for MetricsEvaluator API (CUDA ≥ 11.3).
+  +++ Fixed `cuptid_init` return value and potential overflows.
```

Comment:
For `ROCM/ROCP_SDK`, you have the "+++ Added multiple libhsa search paths". 

If you wanted to, you could mention something similar for `CUDA` and `NVML` as PR #347 introduced dynamically searching for the Cuda shared objects. An example:

```
CUDA:
+++ Introduced dynamically searching for the shared objects libcuda, libcudart, libnvperf_host, and `libcupti`.

NVML:
+++ Introduced dynamically searching for the shared object libnvidia-ml.
```